db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 20_220_222_200_721) do
+ActiveRecord::Schema[7.0].define(version: 20_220_224_171_313) do
   # These are extensions that must be enabled in order to support this database
   enable_extension 'plpgsql'
 
```

Comment:
- [ ] The table `blogs` is not defined in the given Entity Relation Diagram. Kindly ensure to create only the provided tables in the ERD.

Reply:
**@ Fellow Code Reviewers:**

Kindly approve this project. This project has fulfilled the requirements. Thanks

Reply:
@rloterh, thank you and it was nice speaking with you. 

As I have explained, the blog table represents the post table in the ERD. Changing the name of the table at this point will generate a lot of issues because of the instances already installed.

Cheer!

I understand you thought it is an additional table that is not mentioned in the ERD.